src/core/environment.js

```diff
@@ -1152,8 +1152,61 @@ function exitFullscreen() {
  * @example
  * <div>
  * <code>
+ * // Example 2: Convert 2D world coordinates of a rotating square to screen coordinates
+ * function setup() {
+ *   createCanvas(400, 400);
+ *   let vertices = [
+ *     createVector(-10, -10),
+ *     createVector(10, -10),
+ *     createVector(10, 10),
+ *     createVector(-10, 10)
+ *   ];
+ *
+ *   push();
+ *   translate(200, 200);
+ *   rotate(PI / 4);
+ *   let screenPos = vertices.map(v => worldToScreen(v));
+ *   pop();
+ *
+ *   background(200);
+ *   screenPos.forEach((pos, i) => {
+ *     text(`(${pos.x.toFixed(1)}, ${pos.y.toFixed(1)})`, pos.x, pos.y);
+ *   });
+ * }
  * </code>
  * </div>
+ * @example
+ * <div>
+ * <code>
+ * // Example 1: Convert 3D world coordinates of a rotating cube to 2D screen coordinates
+ * function setup() {
+ *   createCanvas(400, 400, WEBGL);
+ *   let vertices = [
+ *     createVector(-50, -50, -50),
+ *     createVector(50, -50, -50),
+ *     createVector(50, 50, -50),
+ *     createVector(-50, 50, -50),
+ *     createVector(-50, -50, 50),
+ *     createVector(50, -50, 50),
+ *     createVector(50, 50, 50),
+ *     createVector(-50, 50, 50)
+ *   ];
+ *
+ *   push();
```

Comment:
How do you feel about moving this to `draw()` and making the rotation change slowly over time?

Reply:
A great idea  , I must say!